Gemfile

```diff
@@ -0,0 +1,48 @@
+source 'https://rubygems.org'
+
+ruby '3.3.0'
+
+gem 'bcrypt'
+gem 'bootsnap', require: false
+gem 'cancancan'
+gem 'cocoon'
+gem 'devise'
+gem 'doorkeeper'
+gem 'dotenv-rails'
+gem 'faker'
+gem 'grape'
+gem 'grape-entity'
+gem 'i18n'
+gem 'importmap-rails'
+gem 'jbuilder'
+gem 'jquery-rails'
+gem 'pg'
+gem 'phony_rails'
+gem 'pry'
+gem 'puma', '>= 5.0'
+gem 'rack'
+gem 'rails', '~> 7.1.3', '>= 7.1.3.4'
+gem 'rubocop'
+gem 'rubocop-rails'
```

Comment:
Are these used in any environments other than development?

Reply:
Yes bhai, for maintaining the same standard, we need it for the whole project.